src/RwCalculators/GReWeighthA2025.cxx

```diff
@@ -0,0 +1,299 @@
+//____________________________________________________________________________
+/*
+ Copyright (c) 2003-2018, The GENIE Collaboration
+ For the full text of the license visit http://copyright.genie-mc.org
+
+ Authors: Costas Andreopoulos <costas.andreopoulos \at stfc.ac.uk>
+          University of Liverpool & STFC Rutherford Appleton Lab
+
+          Jim Dobson <J.Dobson07 \at imperial.ac.uk>
```

Comment:
Authors are Jim Dobson and Costas, but Mohamed Saad (@M7medS3d) is making the pull request; should the list of authors be expanded?